app/com/arpnetworking/metrics/portal/scheduling/impl/BaseSchedule.java

```diff
@@ -149,7 +153,7 @@ public B setRunUntil(@Nullable final Instant runUntil) {
 
         @SuppressFBWarnings(value = "UPM_UNCALLED_PRIVATE_METHOD", justification = "invoked reflectively by @ValidateWithMethod")
         private boolean validateRunAtAndAfter(final Instant runAtAndAfter) {
-            return (_runUntil == null) || !runAtAndAfter.isAfter(_runUntil);
+            return !runAtAndAfter.equals(Instant.MIN) && ((_runUntil == null) || !runAtAndAfter.isAfter(_runUntil));
```

Comment:
This is one of the places where I'd rather the validator require you to throw an exception because at least then the validation you're trying to uphold is obvious with an error message